README.md

```diff
@@ -1351,6 +1345,10 @@ drive report-issue -title "Can't open my file" -body "Drive trips out every time
 cat bugReport.txt | drive issue -piped -title "push: dump on pushing from this directory"
 ```
 
+### Automating
```

Comment:
"curated" sounds a little... strange. I'm not sure if all people understand what that means.